drivers/SmartThings/zigbee-presence-sensor/src/aqara/init.lua

```diff
@@ -0,0 +1,132 @@
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy ofF the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local cluster_base = require "st.zigbee.cluster_base"
+local data_types = require "st.zigbee.data_types"
+local PresenceSensor = capabilities.presenceSensor
+local MovementSensor = capabilities["stse.movementSensor"]
+
+local PRIVATE_CLUSTER_ID = 0xFCC0
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+local MFG_CODE = 0x115F
+
+local MONITORING_MODE = 0x0144
+local RESET_MODE = 0x0157
+
+local MOVEMENT_TIMER = "movement_timer"
+local MOVEMENT_TIME = 5
+
+local SENSITIVITY = "stse.sensitivity"
+local RESET_PRESENCE = "stse.resetPresence"
+local APP_DISTANCE = "stse.approachDistance"
+
+local FINGERPRINTS = {
+  { mfr = "aqara", model = "lumi.motion.ac01" }
+}
+
+local is_aqara_products = function(opts, driver, device, ...)
+  for _, fingerprint in ipairs(FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
```

Comment:
small nitpick here, but since there's only one fingerprint using this sub-driver, you could just do

`return device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model`

if you re-write `FINGERPRINTS` to just be `fingerprint = {mfr = "...", model = "..."}`

Reply:
I'll fix it.